python/cuML/kmeans/c_kmeans.pxd

```diff
@@ -1,5 +1,4 @@
 import numpy as np
-cimport numpy as np
 np.import_array
```

Comment:
What behavior will this have without cimport numpy? I think we can remove this.

Reply:
Sublime find all didn't find this instances in my computer :P